data/src/main/java/com/bottlerocketstudios/brarchitecture/data/model/AuthorDto.kt

```diff
@@ -9,5 +9,10 @@ import kotlinx.parcelize.Parcelize
 @Parcelize
 data class AuthorDto(
     @Json(name = "user") val userInfo: UserDto?,
-    @Json(name = "type") val type: String?
-) : Parcelable, Dto
+    @Json(name = "display_name") val displayName: String?,
+    @Json(name = "links") val links: PrLinksDto?,
+    @Json(name = "type") val type: String?,
+    @Json(name = "uuid") val uuid: String?,
+    @Json(name = "account_id") val accountId: String?,
+    @Json(name = "nickname") val nickName: String?,
+) : Parcelable
```

Comment:
I think this should keep the Dto parent class as a "tag" of sorts.

Reply:
Also, could you take a few minutes to verify if the Parcelable tag is needed?  No need to generate code for that if we aren't parceling.

Reply:
@BottleRocket-Colin deleting Parcelable created a ripple effect across several Dto responses associated with more than just PR screen. I'm concerned about removing it since i'd have to every where else too